build_docs.py

```diff
@@ -995,12 +995,13 @@ def symlink(
     directory_path = path / directory
     if not directory_path.exists():
         return  # No touching link, dest doc not built yet.
-    if link.exists() and readlink(link) == directory:
-        return  # Link is already pointing to right doc.
-    if link.exists():
-        link.unlink()
-    link.symlink_to(directory)
-    run(["chown", "-h", ":" + group, str(link)])
+
+    if not link.exists() or readlink(link) != directory:
+        # Link does not exist or points to the wrong target.
+        if link.exists():
+            link.unlink()
```

Comment:
I believe this could be simplified further by replacing these two lines with `link.unlink(missing_ok=True)`.